### PR TITLE
Typos and Grammar Changes

### PR DESCRIPTION
**Pull request by remedcu**
Things changed:
- Some spelling
- Some grammar based changes

### CODE REVIEW

**Comment by d-xo** [comment]
Thanks for the PR. this is a little too extensive and opinionated imo, and I'm not really into a lot of the grammar changes, I would be happy to merge a more focused change that fixes spelling mistakes only.

**Comment by remedcu** [comment]
@d-xo thanks for a quick reply. Updated with mostly typos.

File: readme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ developers in the wild.
 
 This makes building smart contracts that interface directly with ERC20 tokens challenging to say the
 least, and smart contract developers should in general default to the following patterns when
-interaction with external code is required:
```
Comment (d-xo) [inline readme.md:14]: I prefer the original version

File: readme.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ extortion attempt against users of the blocked contract.
 
 Some tokens can be paused by an admin (e.g. `BNB`, `ZIL`).
 
-Similary to the blocklist issue above, an admin controlled pause feature opens users
```
Comment (d-xo) [inline readme.md:142]: I prefer the original

Comment (remedcu) [reply readme.md]: Reverted

**Pull request merged**